lib/autocomplete.ts

```diff
@@ -4,18 +4,23 @@ import {IOpenController, IPopupContent, PopupControl, setPopupToFunc} from './po
 
 /**
  * IAutocomplete options adds two properties IMenuOptions to customize autocomplete behavior:
- *
- * contentFunc: Overrides the passed-in optionArray to allow creating menu rows
- *   with special behavior when selected.
- *
- * findMatch: Overrides default case-insensitive row select behavior.
  */
 export interface IAutocompleteOptions extends IMenuOptions {
-  contentFunc?: (ctl: IOpenController) => DomElementArg[];
+
+  // Overrides default case-insensitive row select behavior.
   findMatch?: (content: HTMLElement[], value: string) => HTMLElement|null;
+
+  // If true, the value is updated when a choice is selected (ie: by mouse over or arrow
```

Comment:
I don't think we ever want to update value on mouseover, and always want on arrow navigation. Are you sure this option is desirable?

Reply:
Nope, I thought it was better to have arrow navigation and mouse over have the same behavior, but I don't have a strong opinion about it. Ok to remove, to keep mouseover as it was and only change arrow navigation.